ansible/roles/prerun-job/tasks/main.yaml

```diff
@@ -0,0 +1,60 @@
+---
+
+#- name: Install required Ansible collections
+#  delegate_to: localhost
+#  shell: |
+#    ansible-galaxy collection install ibm.blockchain_platform
+#    ansible-galaxy collection install kubernetes.core
+
+- name: register temporary directory
+  tempfile:
+    state: directory
+  register: tmp_directory
+
+- name: Configure awscli credentials for IAM user
+  shell: /usr/local/bin/aws configure set {{ item.key }} {{ item.value }} --profile cicd-user
+  no_log: false
+  with_dict:
+    aws_access_key_id: "{{ aws_access_key }}"
+    aws_secret_access_key: "{{ aws_secret_key }}"
+    region: "{{ aws_region }}"
+    format: "table"
+    external_id: "{{ aws_external_id }}"
+  changed_when: false
+
+- name: Configure awscli credentials for IAM role
+  shell: /usr/local/bin/aws configure set {{ item.key }} {{ item.value }} --profile cicd-role
+  no_log: false
+  with_dict:
+    external_id: "{{ aws_external_id }}"
+    source_profile: "cicd-user"
+    role_arn: "{{ aws_iam_role }}"
+    region: "{{ aws_region }}"
+  changed_when: false
+
+- name: Test AWS CLI is functioning
+  shell: export AWS_PROFILE=cicd-role; /usr/local/bin/aws sts get-caller-identity
+  register: aws_caller_identity
+
+#- name: "Install package dependencies - yum"
```

Comment:
This did not work for some reason. Had to do it manually on the target host.
@ndzambazov-jb can you please review this prerun-job and move away everything related to setup to the setup role.
I also added gcc as a yum package as that is also required for installing the fabric sdk

Reply:
i have moved setup steps to its own setup role - setup/fabric. I am setting up default aws profile as well to avoid `shell: export ...` steps.